oidc-consumer/src/index.ts

```diff
@@ -230,7 +261,7 @@ class OidcConsumer {
    * @param [httpOptions] Optional http options passed through the underlying http library while refreshing token
    * @returns refreshedToken
    */
-  async refresh(token: any, scope?: string, httpOptions?: WreckHttpOptions) {
+  async refresh(token: any, next: NextFunction, scope?: string, httpOptions?: WreckHttpOptions) {
```

Comment:
these are utility methods not called with next
can avoid passing errors to next here